css/index.css

```diff
@@ -66,11 +66,23 @@ p {
     line-height: 1.4;
 }
 
+hr{
+	width: 100%;
+}
+
 .container {
     width: 800px;
     margin: 0 auto;
+		display: flex;
 }
 
+.containerabout{
```

Comment:
Typically it's preferred to use hyphens in mutli-word classes. Look at the CSS that was already here for reference.